Bottom: 8995dca717e96bc6507dcbcf3beef4fddcadc811
Top:    980d5c447476a20e78eff325b469fa47d64df285
Author: scott snyder <sss@karma>
Date:   2020-02-01 10:42:16 -0500

Refresh of unique_warning

---

diff --git a/libstdc++-v3/include/bits/unique_ptr.h b/libstdc++-v3/include/bits/unique_ptr.h
index 131937a8e53..21eb0eb1aaa 100644
--- a/libstdc++-v3/include/bits/unique_ptr.h
+++ b/libstdc++-v3/include/bits/unique_ptr.h
@@ -453,8 +453,8 @@ _GLIBCXX_BEGIN_NAMESPACE_VERSION
       }
 
       // Disable copy from lvalue.
-      unique_ptr(const unique_ptr&) = delete;
-      unique_ptr& operator=(const unique_ptr&) = delete;
+      unique_ptr(const unique_ptr&) = delete; // If you see an error here, you probably forgot to use std::move with unique_ptr
+      unique_ptr& operator=(const unique_ptr&) = delete; // If you see an error here, you probably forgot to use std::move with unique_ptr
   };
 
   /// 20.7.1.3 unique_ptr for array objects with a runtime length
